src/styles/_main-page.scss

```diff
@@ -283,13 +283,14 @@ button {
   top: -10rem;
   right: 1rem;
   float: right;
+
   &::before {
     //edit icon
     content: "";
     height: 2rem;
     width: 2rem;
     display: inline-block;
-    // background-image: url("./icons/png/edit1.png");
+    background-image: url("../icons/png/edit1.png");
```

Comment:
In webpack, the path must be relative to the scss file. That's why is one folder behind in the path `../`